scripts/lib/CIME/XML/env_batch.py

```diff
@@ -402,8 +402,15 @@ def get_submit_args(self, case, job):
                             rval = val
                     else:
                         rval = val
+
+                    # We don't want floating-point data
+                    try:
```

Comment:
py3 bug fix

Reply:
99% sure we'll never want to pass a floating point to the batch submission command. Can you think of a context where we would?